pymcp/middleware.py

```
@@ -0,0 +1,60 @@
+        allow_origins=config.cors["allow_origins"],
+        allow_credentials=config.cors["allow_credentials"],
+        allow_methods=config.cors["allow_methods"],
+        allow_headers=config.cors["allow_headers"],
+    )
+
+    # Compression
+    if config.compression.get("enabled", False):
+        app.add_middleware(GZipMiddleware)
+    # Custom middleware
+    for mw in config.custom:
+        if not hasattr(mw, "__call__"):
```

Ruff (0.11.9)

Using `hasattr(x, "__call__")` to test if x is callable is unreliable. Use `callable(x)` for consistent results.

Replace with `callable()`

(B004)